src/app/link-account/pages/institutional-linking/institutional-linking.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, OnInit } from '@angular/core'
+import { FormGroup } from '@angular/forms'
+import { DiscoService } from '../../../core/disco/disco.service'
+import { OauthService } from '../../../core/oauth/oauth.service'
+import { take } from 'rxjs/operators'
+import { ShibbolethSignInData } from '../../../types/shibboleth-sign-in-data'
+import { Institutional } from '../../../types/institutional.endpoint'
+
+@Component({
+  selector: 'app-institutional-linking',
+  templateUrl: './institutional-linking.component.html',
+  styleUrls: [
+    './institutional-linking.component.scss',
+    './institutional-linking.component.scss-theme.scss',
+  ],
+  host: { class: 'container' },
```

Comment:
The use of this host is against the angular style guide https://angular.io/guide/styleguide#style-06-03)

This was supposed to be caught by the linter but it seems like at some update the migration failed to update some rules.

you can go and change these 3 lines 

https://github.com/ORCID/orcid-angular/blob/46abde39c7a1905bb7ee11c32070025486099368/tslint.json#L88-L90

with this 

```
    "no-inputs-metadata-property": true,
    "no-outputs-metadata-property": true,
    "no-host-metadata-property": true,
```

so the linter catch this thing again

Reply:
More info about the change on this rules names https://github.com/mgechev/codelyzer/issues/791#issuecomment-474031004

Reply:
Ok